src/components/Todo.tsx

```diff
@@ -3,41 +3,28 @@ import '../style.css';
 import DisplayList from "./DisplayList";
 import RemainingListCount from "./RemainingTaskCount"
 import FormInput from './FormInput';
-
-interface TodoListProps {
-    id: number;
-    description: string;
-}
+import TodoListProps from '../TypescriptProps';
 
 interface ListProps {
     list: TodoListProps[];
 }
 
 const Todo: React.FC<ListProps> = ({list}) => { //{list} would replace props so below doesn't look like props.list
-
     const [ toDoList, setToDoList ] = useState(list);
-
     const addTask = (userInput: any ) => {
```

Comment:
userInput should be of type string. Avoid using 'any' where you can , its a typescript cheat ! :)

Reply:
I updated this function type to string. Resolving out.